### PR TITLE
lambda.part1.exercise1(Brovkin)

### PR DESCRIPTION
**Pull request by fedorBrovkin**

### CODE REVIEW

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+
+        Comparator<Person> comparatorPersonByAdge = new Comparator<Person>() {
```
Comment (shaart) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: You should add new class declaration which will implement Comrapator<Person> and use it's instance

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    @Test
+    public void findFirstWithAge30UsingGuavaPredicate() {
+        List<Person> persons = Arrays.asList(getPersons());
+
+        // TODO использовать FluentIterable
+        Person person = null;
+        person = FluentIterable.from(persons)
+            .filter(value->value.getAge()==30).first().get();
```
Comment (shaart) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: You should use Guave Predicate here, not lambda

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 1 above)
Comment (fedorBrovkin) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: Ok. will change it in 10 minutes.

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 2 above)
Comment (fedorBrovkin) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: Done.

**Pull request merged**